### PR TITLE
Support log level query methods

### PR DESCRIPTION
**Pull request by urmastalimaa**
This allows the logger to be used for AR queries.

### CODE REVIEW

File: lib/logasm.rb [hunk 1]
```diff
@@ -38,6 +40,14 @@ def fatal(*args)
     log :fatal, *args
   end
 
+  def method_missing(method, *args)
```
Comment (take-five) [inline lib/logasm.rb]: https://robots.thoughtbot.com/always-define-respond-to-missing-when-overriding

Comment (urmastalimaa) [inline lib/logasm.rb]: yes, thanks, missed that

**Comment by willmore** [comment]
>This allows the logger to be used for AR queries.

Can you elaborate on this? What interface do AR queries need wrt a logger?

**Comment by urmastalimaa** [comment]
It wants a logger that adheres to the standard Ruby logger, more specifically it calls the `debug?` and `info?` methods on the logger.

File: lib/logasm.rb (hunk 1 above)
Comment (urmastalimaa) [inline lib/logasm.rb]: https://github.com/salemove/logasm/commit/0684cbdc0f73d590e9a01cd7e76d8e963a895a50

**Comment by willmore** [comment]
> It wants a logger that adheres to the standard Ruby logger,

Is there an interface that we should be implementing? How do we ensure we stick to the standard?

**Comment by urmastalimaa** [comment]
http://ruby-doc.org/stdlib-2.1.0/libdoc/logger/rdoc/Logger.html, we are curently missing some other functions from it, `#add`, `#close`, `#datetime_format` to be exact.

**Comment by willmore** [comment]
Is there a way we can programmatically assert in tests that all required methods are present?

**Comment by urmastalimaa** [comment]
Yeah, was also thinking about that. I'll add a skipped test for that and create a task to implement missing methods.

**Comment by urmastalimaa** [comment]
https://github.com/salemove/logasm/pull/17

**Pull request merged**